UITests/Global.swift

```diff
@@ -138,8 +138,10 @@ extension KIFUITestActor {
 class BrowserUtils {
     /// Close all tabs to restore the browser to startup state.
     class func resetToAboutHome(tester: KIFUITestActor) {
-        if tester.tryFindingTappableViewWithAccessibilityLabel("Cancel", error: nil) {
+        do {
+            try tester.tryFindingTappableViewWithAccessibilityLabel("Cancel")
             tester.tapViewWithAccessibilityLabel("Cancel")
+        } catch _ {
```

Comment:
Nit: you could do `if try: tester...` if you want to reduce the churn and avoid the empty catch block.